gustaf/show.py

```diff
@@ -49,7 +49,7 @@ def show(*gus_obj, **kwargs):
 
     if vis_b.startswith("vedo"):
         return show_vedo(*gus_obj, **kwargs)
-    elif vis_b.startswith("trimesh"):
+    elif vis_b.startswith("trimesh"):  # noqa: SIM114
```

Comment:
Otherwise, this gives an error that both elif could be handled in a single statement with an or.